Enable GitHub Issues

slawekjaranowski opened the pull request:
Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MINSTALL) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[MINSTALL-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MINSTALL-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [ ] You have run the integration tests successfully (`mvn -Prun-its clean verify`).

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [ ] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [ ] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Bukama reviewed (approved):
Compared with our PRs like this. Only difference is that in some (for exmaple resources) a small PR_automation.yml was also added to label things. But I think it's not a must-have.

slawekjaranowski commented:
> Compared with our PRs like this. Only difference is that in some (for exmaple resources) a small PR_automation.yml was also added to label things. But I think it's not a must-have.

added early #100, #101

hboutemy commented:
we already have many Jira to GH Issues migration started https://cwiki.apache.org/confluence/display/MAVEN/JIRA+to+GitHub+Issues+switching
can we share progress, learnings, remaining TODO before planning addition of new projects to the list, please?

slawekjaranowski commented:
> we already have many Jira to GH Issues migration started https://cwiki.apache.org/confluence/display/MAVEN/JIRA+to+GitHub+Issues+switching can we share progress, learnings, remaining TODO before planning addition of new projects to the list, please?

More of switched project is our internal, not widely used by external users.
I will see how users (and we as maintainers) will work with it.

on mentioned wiki - we have a progress, steps, used tools ... 

I'm going to release projects with GitHub issues to see what we need updated in release procedure
